integrated_channels/api/v1/logs/views.py

```diff
@@ -104,7 +108,18 @@ def get_queryset(self):
             # The serialized `sync_last_attempted_at` value is the representation of the `remote_<X>_at` values
             if "sync_last_attempted_at" in content_sort_by[0]:
                 order = "-" if "-" in content_sort_by[0] else ""
-                content_sort_by = (f"{order}remote_deleted_at", f"{order}remote_updated_at", f"{order}remote_created_at")
+                # Arbitrarily old date to make sure null dates are considered lower in the `Greatest` annotation than any existing or to be
+                # created datetime in the audits
+                long_ago = datetime(year=1995, month=6, day=18)
+                return ContentMetadataItemTransmission.objects.annotate(
+                    newest_date=Greatest(
+                        Coalesce('remote_updated_at', Cast(long_ago, DateTimeField()), output_field=DateTimeField()),
+                        Coalesce('remote_deleted_at', Cast(long_ago, DateTimeField()), output_field=DateTimeField()),
+                        Coalesce('remote_created_at', Cast(long_ago, DateTimeField()), output_field=DateTimeField()),
+                    )
+                ).filter(
+                    **content_metadata_transmission_audit_filter
+                ).order_by(f'{order}newest_date')
```

Comment:
nit: I'm thinking we could encapsulate most of the annotation/sort logic in a helper method
```suggestion
                return SortByDateAggregation(objects=ContentMetadataItemTransmission.objects.filter(**content_metadata_transmission_audit_filter), date_fields=['remote_updated_at','remote_deleted_at','remote_created_at'])
```